unicore/comments/service/tests/test_db.py

```diff
@@ -0,0 +1,49 @@
+import mock
+
+from unicore.comments.service.tests import BaseTestCase
+from unicore.comments.service import db, app
+
+
+class DBTestCase(BaseTestCase):
+
+    def setUp(self):
+        super(DBTestCase, self).setUp()
+        app.db_engine = self.engine
+        app.config = self.config
+
+    def tearDown(self):
+        super(DBTestCase, self).tearDown()
+        del app.db_engine
+        del app.config
```

Comment:
You can also add this with `self.addCleanup(lambda: del app.db_engine)` in the `setUp` if you find that being more convenient.